core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/DataAccessImpl.java

```diff
@@ -383,6 +387,17 @@ protected void initPreparedStatements() {
                 "SELECT tenant_id, type, metric " +
                 "FROM metrics_tags_idx " +
                 "WHERE tenant_id = ? AND tname = ? AND tvalue = ?");
+
+        find5MinStats = session.prepare(
+                "SELECT time, max, min, avg, median, samples, sum, percentiles " +
+                "FROM rollup300 " +
+                "WHERE tenant_id = ? AND metric = ? AND shard = 0");
+
+        find5MinStatsDesc = session.prepare(
+                "SELECT time, max, min, avg, median, samples, sum, percentiles " +
+                "FROM rollup300 " +
```

Comment:
Do we need new table for each rollup? Wouldn't it make sense to have something like "tenant_id, metric, shard, rollup_period" in the partition key?